.evergreen/generate_evergreen_tasks.js

```diff
@@ -529,7 +529,7 @@ SINGLETON_TASKS.push(
       tags: ['lint-checks'],
       commands: [
         updateExpansions({
-          NODE_LTS_VERSION: LOWEST_LTS,
+          NODE_LTS_VERSION: LATEST_LTS,
```

Comment:
I made this change since the updated eslint related dependencies will not work on Node 16 (`structuredClone` isn't available). I figure it was ok to run our lint tasks on latest LTS since it's dev-only and our regular Node 16 are all still passing.

Reply:
So long as we still unit test on lowest, I agree that is fine.